src/test/java/service/activities/QuantityVerifierImplTest.java

```diff
@@ -0,0 +1,38 @@
+package service.activities;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class QuantityVerifierImplTest {
+    private QuantityVerifier quantityVerifier;
+
+    @BeforeEach
+    void setUp() {
+        quantityVerifier = new QuantityVerifierImpl();
+    }
+
+    @Test
+    void quantityVerify_Ok() {
+        String fruit = "apple";
+        int quantity = 100;
+        assertDoesNotThrow(() -> quantityVerifier.quantityVerify(quantity, fruit));
```

Comment:
not fixed
![image](https://github.com/mate-academy/jv-fruit-shop-tests/assets/73041091/19602d72-7d11-4c45-8d51-be70f3b0f585)